crypto/merklearray/array.go

```diff
@@ -16,10 +16,12 @@
 
 package merklearray
 
+import "github.com/algorand/go-algorand/crypto"
+
 // An Array represents a dense array of leaf elements that are
 // combined into a Merkle tree. The Marshal method returns a byte slice that represents the object
 // that the Tree will use to hash the leaves.
```

Comment:
This comment is very vague. Please explain what "represents" means with more details. This is an exported interface, so the code will be much more readable when this interface is well understood.